Gemfile

```diff
@@ -1,7 +1,7 @@
 source 'https://rubygems.org'
 
 # Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
-gem 'rails', '4.0.1'
```

Comment:
You're totally right - I don't think it actually works for Rails though. Take a look at your `.lock` for rails:

`rails`